src/migrations/20240506144701-update-course-model.js

```diff
@@ -0,0 +1,36 @@
+const { prepMigration, removeTables } = require('../lib/migration');
+
+/** @type {import('sequelize-cli').Migration} */
+module.exports = {
+  async up(queryInterface, Sequelize) {
+    await queryInterface.sequelize.transaction(async (transaction) => {
+      const sessionSig = __filename;
+      await prepMigration(queryInterface, transaction, sessionSig);
+
+      // add persistsOnUpload to the Courses table
+      await queryInterface.addColumn('Courses', 'persistsOnUpload', {
+        type: Sequelize.BOOLEAN,
+        allowNull: false,
+        defaultValue: false,
+      }, { transaction });
+
+      // insert an "other" course
+      await queryInterface.bulkInsert('Courses', [
```

Comment:
Not that it matters but should we be using bulk insert for a single entry?

Reply:
yeah its the easiest way to do an insert via the ORM in a migration. evaluates to the same thing.